spec/requests/social_previews_spec.rb

```diff
@@ -141,4 +142,32 @@
       expect(response).to redirect_to(image_url)
     end
   end
+
+  describe "GET /social_previews/comment/:id" do
+    it "renders proper comment name" do
+      get "/social_previews/comment/#{comment.id}"
+      expect(response.body).to include CGI.escapeHTML(comment.title)
+    end
+
+    it "renders associated article name" do
+      get "/social_previews/comment/#{comment.id}"
+      expect(response.body).to include CGI.escapeHTML(comment.commentable.title)
+    end
+
+    it "renders consistent HTML between requests" do
+      # We use the HTML for caching. It needs to be deterministic (if data is unchanged, the HTML should be the same)
+      get "/social_previews/comment/#{comment.id}"
+      first_request_body = response.body
+
+      get "/social_previews/comment/#{comment.id}"
+      second_request_body = response.body
+
+      expect(first_request_body).to eq second_request_body
+    end
+
+    it "renders and image when requested and redirects to iamge url" do
```

Comment:
small typo, `iamge` instead of `image`

Reply:
Sorry for missing out on that! Also updated another instance of the typo on line 140